CMakeLists.txt

```diff
@@ -33,10 +32,12 @@ if ((NOT is_multi_config) AND (NOT CMAKE_BUILD_TYPE))
     set(CMAKE_BUILD_TYPE Release)
 endif ()
 
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_CURRENT_SOURCE_DIR}/cmake/Modules/")
```

Comment:
This can also be set from outside on cmake command. We should not overwrite and lose that.

Reply:
From CMake [docs for CMAKE_MODULE_PATH](https://cmake.org/cmake/help/latest/variable/CMAKE_MODULE_PATH.html):
```
... By default it is empty, it is intended to be set by the project. 
```

This is very common. See also: https://gitlab.kitware.com/cmake/community/-/wikis/doc/tutorials/How-To-Find-Libraries